### ISSUE TITLE
Webplayer

### ISSUE THREAD

**Issue by anon97945** [odysseusmax/tg-index#16]
Webplayer can't playback when skipped to time. It only can stream from beginning.

**Comment by odysseusmax** [comment]
Please give more information. Which browser, what kind of video, etc...

**Comment by pseudokawaii** [comment]
@odysseusmax perhaps he's talking about this ~

https://user-images.githubusercontent.com/63403140/121785980-ec25a380-cbde-11eb-83ff-eaf592f1a596.mp4

**Comment by rayanfer32** [comment]
Limitations of fluid player i guess 🤔

**Comment by pseudokawaii** [comment]
@rayanfer32 maybe it can't seek more than what it has pre-rolled or maybe it's a limitation with telegram's API, they be already giving way more than what one could hope from just a messenger. a stricter bandwidth cap or requests limitation is to be expected.

**Comment by rayanfer32** [comment]
But from what I know... Seeking should work as it's working in vlc player.
I'll push some new features and fixes tomorrow  which will allow to play on
external player using direct m3u export.

**Comment by pseudokawaii** [comment]
@rayanfer32 yeah, seeking works fine if you copy the download link of the media and stream it using any media player like VLC. Aside from that, even downloading has some issues. Only single connection to the server is supported (maybe it's a repl.it limitation), if i try to download a file using IDM with multiple simultaneous connections enabled, the download becomes corrupted and fails in the midway.

**Comment by rayanfer32** [comment]
Oh i haven't faced it yet ... Will test this limitation and let you know.

**Comment by dgbkn** [comment]
https://github.com/dgbkn/TG-FileStreamBot-tgindex-final/blob/main/WebStreamer/server/stream_routes.py
See this in this bot's code mp4 streaming on the web works like a charm, please fix this issue on this indexing app

**Comment by dgbkn** [comment]
btw i have fixed it if anyone wants it let me know

**Comment by dgbkn** [comment]
in my repo which is forked with this

**Comment by rayanfer32** [comment]
Yes please share it

**Comment by pseudokawaii** [comment]
@dgbkn would you mind giving a link to that fork? I searched your profile but didn't find it.

**Comment by rayanfer32** [comment]
> @dgbkn would you mind giving a link to that fork? I searched your profile but didn't find it.

I think he is talking about this repo and this fix https://github.com/dgbkn/TgindexPro-telegram-SuperFinal/commit/18bb6802dfc2387d3060d883b4943867eb61b6c2
Use of hard-coded values for part_size

**Comment by dgbkn** [comment]
yes bro

**Comment by dgbkn** [comment]
check its code
i have changed two files

**Comment by dgbkn** [comment]
now its working ??

**Comment by dgbkn** [comment]
@rayanfer32  i have created a pull request in your forked repo

**Comment by dgbkn** [comment]
check it now its working like i charm

**Comment by dgbkn** [comment]
> > @dgbkn would you mind giving a link to that fork? I searched your profile but didn't find it.
> 
> I think he is talking about this repo and this fix [dgbkn/TgindexPro-telegram-SuperFinal@18bb680](https://github.com/dgbkn/TgindexPro-telegram-SuperFinal/commit/18bb6802dfc2387d3060d883b4943867eb61b6c2)
> Use of hard-coded values for part_size

NO I have changed more
see the pull request that i have created

**Comment by dgbkn** [comment]
also added CROS support and fixed Downloads Resuming <>

**Comment by dgbkn** [comment]
![image](https://user-images.githubusercontent.com/65810521/125923081-9c39bd78-cf62-4091-8857-45193365a9a4.png)

**Comment by dgbkn** [comment]
see this image its perfectly streaming <>

**Comment by dgbkn** [comment]
@rayanfer32  found that pull request ??

**Comment by pseudokawaii** [comment]
@dgbkn can you stop spamming and let him check?

**Comment by dgbkn** [comment]
ok i'll wait

**Comment by dgbkn** [comment]
now it is working ??? @rayanfer32

**Comment by imner0** [comment]
> now it is working ??? @rayanfer32

can you help me to deploy locally on vps
